docs/policy/service-migrations-spring-2018.md

```diff
@@ -92,6 +92,11 @@ In the future, we will use the following CA certificate services:
 |                               | LHC users should continue to request their user certificates from CERN.             |
 | Web-Based services            | [Let's Encrypt](https://letsencrypt.org)                                            |
 
+!!! note
+    The semantics of Let's Encrypt certificates are different from those of previous CAs.
+    Please see <https://opensciencegrid.github.io/security/LetsEncryptOSGCAbundle/> for the security and
+    setup implications of switching to a Let's Encrypt host or service certificate.
```

Comment:
Instead of the plain URL, let's call it the `security team's position on Let's Encrypt`